rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -1726,6 +1726,34 @@ void should_CancelTask_when_CallingCancelEndpoint() {
     }
   }
 
+  @Nested
+  @TestInstance(Lifecycle.PER_CLASS)
+  class TerminateTasks {
+    @Test
+    void should_TerminateTask_when_CallingCancelEndpoint() {
+      String url =
+          restHelper.toUrl(RestEndpoints.URL_TASKS_ID, "TKI:100000000000000000000000000000000000");
+      HttpEntity<Object> auth = new HttpEntity<>(RestHelper.generateHeadersForUser("admin"));
+
+      // retrieve task from Rest Api
+      ResponseEntity<TaskRepresentationModel> responseGet =
+          TEMPLATE.exchange(url, HttpMethod.GET, auth, TASK_MODEL_TYPE);
+
+      assertThat(responseGet.getBody()).isNotNull();
+      TaskRepresentationModel theTaskRepresentationModel = responseGet.getBody();
+      assertThat(theTaskRepresentationModel.getState()).isEqualTo(TaskState.CLAIMED);
+
+      // cancel the task
+      String url2 =
+          restHelper.toUrl(
+              RestEndpoints.URL_TASKS_ID_TERMINATE, "TKI:000000000000000000000000000000000103");
+      responseGet = TEMPLATE.exchange(url2, HttpMethod.POST, auth, TASK_MODEL_TYPE);
+
+      assertThat(responseGet.getBody()).isNotNull();
+      assertThat(responseGet.getBody().getState()).isEqualTo(TaskState.TERMINATED);
```

Comment:
please adjust the instance variables + comment to match with the terminate operation